lib/ReactGridLayout.jsx

```diff
@@ -348,6 +348,13 @@ export default class ReactGridLayout extends React.Component<Props, State> {
     return null;
   }
 
+  shouldComponentUpdate(nextProps: Props, nextState: State) {
+    return (
+      !isEqual(this.props, nextProps) ||
```

Comment:
we really want to compare entire props object here? Maybe we can compare certain props only?

Reply:
With any changes in props, we must rerender the component. We only check the input and only some internal state